mods/cnc/maps/scb30ea/scb30ea-AI.lua

```diff
@@ -0,0 +1,321 @@
+	GDINuke4, GDIDef5, GDIDef6, GDINuke5, GDISilo1, GDISilo2, GDIDef7, GDIDef8, GDISilo3, GDISilo4, GDIDef9 }
+GDIRebuildList = { GDInuke1, GDIproc1, GDIpyle, GDIweap, GDIproc2, GDInuke2, GDInuke3, GDIdef1, GDIdef2, GDIdef3, GDIdef4,
+	GDInuke4, GDIdef5, GDIdef6, GDInuke5, GDIsilo1, GDIsilo2, GDIdef7, GDIdef8, GDIsilo3, GDIsilo4, GDIdef9 }
+BSizes = { nuk2 = CVec.New(2,3), proc = CVec.New(3,4), pyle = CVec.New(2,3), weap = CVec.New(3,3), atwr = CVec.New(1,1), gtwr = CVec.New(1,1), silo = CVec.New(2,1), hpad = CVec.New(2,2) }
+
+ProductionBuildings = { i = GDIPyle, v = GDIWeap, a = GDICYard }
+
+ProdQueue = { i = { }, v = { }, a = { } }
+NewTeam = { }
+TeamJob = "attack"
+AT1 = { "e2", "e2", "e2", "e3", "e3", "v", "jeep", "jeep", "apc" }
+AT2 = { "e2", "e2", "e2", "e3", "e3", "e1", "e1", "e1", "e1" }
+AT3 = { "v", "mtnk", "mtnk", "jeep", "jeep" }
+AT4 = { "e3", "e3", "e1", "e1", "e1", "v", "mtnk", "mtnk", "msam" }
+AT5 = { "v", "jeep", "jeep", "jeep", "apc", "apc" }
+AT6 = { "e3", "e3", "e1", "e1", "e1", "v", "htnk" }
+AT7 = { "v", "htnk", "htnk", "mtnk" }
+AT8 = { "v", "htnk", "htnk", "msam" }
+AT9 = { "v", "mtnk", "msam", "apc", "apc" }
+AT10 = { "e2", "e2", "e2", "e2", "e2", "e2", "e2", "e2", "e2", "e2" }
+OT1 = { "a", "orca", "orca" }
+OT2 = { "a", "orca", "orca", "orca", "orca" }
+AttackTeams = { AT1, AT2, AT3, AT4, AT5, AT6, AT7, AT8, AT9, AT10 }
+
+PT1 = { "v", "htnk" }
+PT2 = { "v", "mtnk", "mtnk" }
+PT3 = { "e2", "e2", "e2", "e3", "e3" }
+PatrolTeams = { PT1, PT2, PT3 }
+
+DT1 = { "v", "mtnk", "mtnk", "msam", "msam" }
+DT2 = { "e3", "e3", "e1", "e1", "e1", "v", "mtnk", "mtnk", "msam" }
+DT3 = { "v", "htnk", "htnk", "msam" }
+DefenseTeams = { DT1, DT2, DT3 }
+
+AddOrcasTo = { AT1, AT2, AT5, AT6, AT9, DT1, DT2, DT3 }
+
+AP1 = { waypoint5.Location, waypoint9.Location, waypoint8.Location, waypoint13.Location }
+AP2 = { waypoint2.Location, waypoint5.Location, waypoint4.Location, waypoint6.Location }
+AP3 = { waypoint2.Location, waypoint3.Location, waypoint4.Location, waypoint6.Location }
+AP4 = { waypoint5.Location, waypoint9.Location, waypoint14.Location, PlayerStart.Location, waypoint13.Location }
+AP5 = { waypoint5.Location, waypoint9.Location, waypoint6.Location }
+AttackPaths = { AP1, AP2, AP3, AP4, AP5 }
+
+InitAi = function()
+
+	RepairNamedActors(GDI, 0.75)
+
+	if Difficulty == "hard" then
+		PBDelay = 115
+		PUDelay = 30
+		ICDelay = 800
+		ProdCDSecs = 30
+		GDI.Resources = 10000
+		minHarvs = 3
+		ProcUpgrade = "AIHProcUpgrade"
+	end
+
+	if Difficulty == "normal" then
+		PBDelay = 140
+		PUDelay = 50
+		ICDelay = 1000
+		ProdCDSecs = 40
+		GDI.Resources = 7000
+		minHarvs = 2
+		ProcUpgrade = "AINProcUpgrade"
+	end
+
+	if Difficulty == "easy" then
+		PBDelay = 180
+		PUDelay = 70
+		ICDelay = 1200
+		ProdCDSecs = 50
+		GDI.Resources = 5000
+		minHarvs = 2
+		Nod.Resources = 3000
+	end
+
+	if ProcUpgrade then ProcUpg = Actor.Create(ProcUpgrade, true, { Owner = GDI } ) end
```

Comment:
Ah this is my bad, sorry. The line I was commenting on was an assignment and there we have a space before and after the braces, but there is no space between them and parenthesis:
```suggestion
	if ProcUpgrade then
		ProcUpg = Actor.Create(ProcUpgrade, true, { Owner = GDI })
	end
```
The other usages seem fine, so I'll quickly mark the places I spotted so you don't have to go over them again.